### ISSUE TITLE
sections retrieved with the wiki API have different formating than content sections

### ISSUE THREAD

**Issue by kroeliebuschie** [goldsmith/Wikipedia#168]
The module is able to retrieve the table of content from the wikipedia API. A very basic functionality of this module is query the table of content and go over the sections. For example: 

```
import wikipedia

wikipedia.set_lang("nl")
page = wikipedia.page("Samenstelling_Tweede_Kamer_1875-1879")

secs = page.sections         #Get a complete list of the page's sections
print(page.section(secs[1])) 
print(page.section(secs[2])) #empty!
 ```

In this example the second section appears to be empty. This is due to the difference in data formatting of `wikipedia.sections` and `wiki.section`. 

```
https://nl.wikipedia.org/w/api.php?action=parse&prop=sections&page=Samenstelling_Tweede_Kamer_1875-1879&format=json
https://nl.wikipedia.org/w/api.php?explaintext=&rvprop=ids&titles=Samenstelling_Tweede_Kamer_1875-1879&prop=extracts|revisions&action=query&format=json
```
These are the actual queries made, the first one returns the sections in their plane formatting and the second returns the page content in wiki formatting. This causes the names retrieved by `wikipedia.sections` to differ from the section names in the `wikipedia.content`-content.

**Comment by kroeliebuschie** [comment]
more details on the problem: https://phabricator.wikimedia.org/T185472

**Comment by barrust** [comment]
Yes, this is an issue with the way the MediaWiki api returns the section titles. I fixed this in a different MediaWiki API wrapper (https://github.com/barrust/mediawiki) which is still maintained. This fix will be in the next release.
